### PR TITLE
Some more cleanup

### PR DESCRIPTION
**Pull request by spk**

### CODE REVIEW

File: app/models/dependency.rb [hunk 1]
```diff
@@ -88,7 +93,7 @@ def to_s
   end
 
   def clean_requirements
-    requirements.gsub /#<YAML::Syck::DefaultKey[^>]*>/, "="
+    requirements.gsub(/#<YAML::Syck::DefaultKey[^>]*>/, "=")
```
Comment (simi) [inline app/models/dependency.rb]: This `gsub` is also doubled in this class and can be DRY'ed IMHO.

File: app/models/dependency.rb [hunk 2]
```diff
@@ -137,4 +142,8 @@ def parse_gem_dependency
   def push_on_to_list
     $redis.lpush(Dependency.runtime_key(self.version.full_name), self.to_s) if self.scope == 'runtime'
   end
+
+  def clean_yaml_sync(reqs)
```
Comment (simi) [inline app/models/dependency.rb]: `clean_yaml_sync` :question:

What about something like this (untested):

```ruby
def clean_requirements(reqs = requirements)
  reqs.gsub(/#<YAML::Syck::DefaultKey[^>]*>/, "=")
end
...
def update_resolved(rubygem)
...
  self.requirements = clean_requirements(reqs)
```

Comment (spk) [inline app/models/dependency.rb]: @simi yes better thanks :exclamation:

File: app/models/dependency.rb [hunk 3]
```diff
@@ -12,8 +12,13 @@ class Dependency < ActiveRecord::Base
 
   attr_accessor :gem_dependency
 
+  def self.unresolved(rubygem)
+    where(:unresolved_name => nil, :rubygem_id => rubygem.id)
```
Comment (arthurnn) [inline app/models/dependency.rb]: can you make this use new Hash syntax? 
If you are touching this lines already, I think is a good idea to update it to new syntax.

Comment (simi) [inline app/models/dependency.rb]: :+1:

**Comment by spk** [comment]
@arthurnn I will remember that for future changes :exclamation:

**Comment by arthurnn** [comment]
thanks

**Comment by simi** [comment]
:clap:

**Pull request merged**